executor/executor_test.go

```diff
@@ -178,6 +178,17 @@ func (s *testSuite) TestAdmin(c *C) {
 	c.Assert(row.GetString(1), Equals, historyJobs[0].State.String())
 	c.Assert(err, IsNil)
 
+	// show DDL job queries test
```

Comment:
please refine these tests by creating a new store.

Reply:
already added tests.

Reply:
Where did we use a new store in this test?
If we don't use a new store, I think the result in line 193 will be unstable.